services/app-graphql/src/schema.graphql

```diff
@@ -833,68 +939,163 @@ type ActuaryContact  {
   actuarialFirmOther: String
 }
 
+"""
+A package in the system 
+that shares a rate with another package.
+It's used as a part of RateFormData
+"""
 type PackageWithSameRate  {
   packageName: String!
   packageId: String!
   packageStatus: String
 }
 
+"""
+RateFormData represents the form data that was inputed by the state
+This type is used for the form data field found on a rate revision
+"""
 type RateFormData {
+    """
+    Can be 'NEW' or 'AMENDMENT'
+    Refers to whether the state is submitting a brand new rate amendment
+    or an amendment to an existing rate certification
+    """
     rateType: RateType
+    """
+    Can be 'RATE_CELL' or 'RATE_RANGE'
+    These values represent how the capitation rate is determined 
+    """
     rateCapitationType: RateCapitationType
+    """
+    Signed certification documents the state uploads 
+    Files can be PDF, DOC, or DOCX format
+    """
     rateDocuments: [GenericDocument!]!
+    """
+    Additional documents the state uploads to support a rate cert
+    Files can be PDF, DOC, DOCX, XLSX, CSV format
+    """
     supportingDocuments: [GenericDocument!]!
+    """
+    The start date of the rating period for a new certification
+    Only relevant if rate type is NEW
```

Comment:
This isn't quite right. We record rateDateStart/End for both NEW and AMENDMENT rates.
We only record amendmentEffectiveDateStart/End for AMENDMENT rates

For NEW rates, rateDateStart/End is the "Rating period" for that NEW rate
for AMENDMENT rates, rateDateStart/End is the "Rating period of original rate certification"
and amendmentEffectiveDateStart/End is the "Effective dates of rate amendment"